### PR TITLE
Add new story into Chinese version

### PR DESCRIPTION
**Pull request by murasak**
please review this. @casey-collab

### CODE REVIEW

File: robot/robot.md [hunk 1]
```diff
@@ -0,0 +1 @@
+500年后醒来，他变成了HALO中的超级战士，需要帮助人类拯救地球。
```
Comment (casey-collab) [inline robot/robot.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by murasak** [comment]
check this out. @casey-collab

File: robot/robot.md (hunk 1 above)
Comment (casey-collab) [inline robot/robot.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**